lib/plugins/rest.js

```diff
@@ -57,7 +63,13 @@ exports.register = function (server, options, next) {
 				return awsAdapter.runInstances(instance, instanceOptions);
 			})
 			.then(function (instance) {
-				reply(instance).created("/v1/instances/" + instance.id);
+				if (instance.keyLocation) {
+					reply(instance).created("/v1/instances/" + instance.id)
+					.header("privatekeylocation", instance.keyLocation);
```

Comment:
Also, since this header is specific to our application, it's common practice to prefix it with `X-`.